src/main/java/org/jabref/gui/openoffice/StyleSelectDialogView.java

```diff
@@ -0,0 +1,132 @@
+
+    public StyleSelectDialogView(DialogService dialogService, StyleLoader loader) {
+        this.dialogService = dialogService;
+        this.loader = loader;
+
+        ViewLoader.view(this)
+                  .load()
+                  .setAsDialogPane(this);
+
+        setResultConverter(button -> {
+            if (button == ButtonType.OK) {
+                viewModel.storePrefs();
+                return tvStyles.getSelectionModel().getSelectedItem().getStyle();
+            }
+            return null;
+
+        });
+    }
+
+    @FXML
+    private void initialize() {
+
+        viewModel = new StyleSelectDialogViewModel(dialogService, loader, preferencesService);
+
+        preview = new PreviewPanel(null, new BibDatabaseContext(), preferencesService.getKeyBindingRepository(), preferencesService.getPreviewPreferences(), dialogService, ExternalFileTypes.getInstance());
+        preview.setEntry(TestEntry.getTestEntry());
+
+        vbox.getChildren().add(preview);
+
+        colName.setCellValueFactory(cellData -> cellData.getValue().nameProperty());
+        colJournals.setCellValueFactory(cellData -> cellData.getValue().journalsProperty());
+        colFile.setCellValueFactory(cellData -> cellData.getValue().fileProperty());
+        colDeleteIcon.setCellValueFactory(cellData -> cellData.getValue().internalStyleProperty());
+
+        new ValueTableCellFactory<StyleSelectItemViewModel, Boolean>().withGraphic(internalStyle -> {
+            if (!internalStyle) {
+                return IconTheme.JabRefIcons.DELETE_ENTRY.getGraphicNode();
+            }
+            return null;
+
+        }).withOnMouseClickedEvent(item -> {
+            return evt -> viewModel.deleteStyle();
+        }).install(colDeleteIcon);
+
+        edit.setOnAction(e -> viewModel.editStyle());
+
+        new ViewModelTableRowFactory<StyleSelectItemViewModel>()
+                                                                .withOnMouseClickedEvent((item, event) -> {
+                                                                    if (event.getClickCount() == 2) {
+                                                                        viewModel.viewStyle(item);
+                                                                    }
+                                                                }).withContextMenu(item -> createContextMenu())
+                                                                .install(tvStyles);
+        tvStyles.getSelectionModel().selectedItemProperty().addListener((observable, oldvalue, newvalue) -> {
+            if (newvalue == null) {
+                viewModel.selectedItemProperty().setValue(oldvalue);
+            } else {
+                viewModel.selectedItemProperty().setValue(newvalue);
+            }
+        });
+
+        tvStyles.setItems(viewModel.stylesProperty());
+
+        add.setGraphic(IconTheme.JabRefIcons.ADD.getGraphicNode());
+
+        EasyBind.subscribe(viewModel.selectedItemProperty(), style -> {
```

Comment:
@NorwayMaple  As you wanted an example for  EasyBind and FXML and javafx, here is one